src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -719,6 +721,38 @@ export class CloudsyncFormComponent implements FormConfiguration {
     this.folderDestinationField = this.fieldSets.config('folder_destination') as FormExplorerConfig;
     this.folderSourceField = this.fieldSets.config('folder_source') as FormExplorerConfig;
     this.formGroup.controls['credentials'].valueChanges.pipe(untilDestroyed(this)).subscribe((res: number | typeof NULL_VALUE) => {
+      if (!res) {
```

Comment:
If I click on the "Add new credential" option the first time, the dialog opens. But if I close the dialog without doing anything, the drop down has "Add new credential" selected and clicking on it again does nothing until you change the value to something else. We should handle that case too where dialog is just closed without success or failure